src/common/contexts/ThemeProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useContext } from 'react';
+
+type Theme = 'light' | 'dark';
```

Comment:
Maybe this can be an `enum` instead of `type`?

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
this has been updated